merlin-server/src/main/java/gov/nasa/jpl/aerie/merlin/server/services/LocalMissionModelService.java

```diff
@@ -270,6 +270,17 @@ public void refreshActivityTypes(final String missionModelId)
     }
   }
 
+  @Override
+  public void refreshResourceTypes(final String missionModelId){
+    try {
+      //instantiate model
```

Comment:
🔵 Looks like a note-to-self comment that can be removed now

Reply:
Thanks! It's been removed.